doc/services/portability/posix/conformance/index.rst

```diff
@@ -92,7 +92,7 @@ POSIX System Interfaces
     _POSIX_SHARED_MEMORY_OBJECTS, -1,
     _POSIX_SPAWN, -1,
     _POSIX_SPORADIC_SERVER, -1,
-    _POSIX_SYNCHRONIZED_IO, -1,
+    _POSIX_SYNCHRONIZED_IO, 200809L, :kconfig:option:`CONFIG_FDTABLE`
```

Comment:
Similarly, ideally we would have a CONFIG_POSIX_SYNCHRONIZED_IO but the following should be more accurate, since that is the option that gates fs.c.

However, we cannot say this is supported until msync, and fdatasync are also present.

```suggestion
    _POSIX_SYNCHRONIZED_IO, -1, :kconfig:option:`CONFIG_POSIX_FS`
```